src/guides/v2.3/config-guide/cache/two-level-cache.md

```diff
@@ -0,0 +1,62 @@
+---
+group: configuration-guide
+title: Using L2 cache in Magento application
+functional_areas:
+  - Configuration
+  - System
+  - Cache
+---
+
+## Set up an L2 syncronized cache as the base cache adapter
```

Comment:
You shouldn't start a topic with a H2. This is a short topic, so delete this H2 and change the H3 to `**Configuration example:**` (minus the backticks)